tests/basic.test.tsx

```diff
@@ -177,11 +177,14 @@ it('only re-renders if value has changed', async () => {
   type Props = { countAtom: typeof count1Atom; name: string }
   const Counter: React.FC<Props> = ({ countAtom, name }) => {
     const [count, setCount] = useAtom(countAtom)
-    const renderCount = React.useRef(0)
+    const commits = useRef(1)
```

Comment:
However, I'm not sure if I understand how RTL and `findBy*` work. It looks like the initial effect is ignored, and I made `1` initially, which seems totally wrong...

BTW, testing with `findBy*` is handy but not like the typical usage. There would be a better way. (Hope someone chimes in eventually to help on it.)